portal-impl/src/content/Language.properties

```diff
@@ -5041,6 +5041,7 @@ the-selected-entry-has-been-deleted=The selected entry has been deleted.
 the-selected-license-is-used-by-at-least-one-product=The selected license is used by at least one product.
 the-selected-list-no-longer-exists=The selected list no longer exists.
 the-selected-role-cannot-be-deleted-because-it-is-a-required-system-role=The selected role cannot be deleted because it is a required system role.
+the-selected-root-folder-is-in-the-recycle-bin-please-remove-it-or-select-another-one=The selected Root Folder is in the Recycle Bin. Please remove it or select another one.
```

Comment:
This should go in portal-language-lang, not in portal-impl. Can you move the key and build lang?

Reply:
Fixed in #2350